Research/HuggingFaceModels/requirements.txt

```
@@ -0,0 +1,11 @@
+datasets
+torch
+transformers[torch]
+transformers
+seqeval
+evaluate
+numpy
+huggingface_hub
+ipywidgets
+pprint
+memory_profiler
```

matplotlib is required for second notebook, not sure why not included
```suggestion
memory_profiler
matplotlib
```

---

I am aware that I said earlier that I will update way of work, but for now, I would wait with that for a team meeting

---

okay, let's leave that for now